package.json

```diff
@@ -27,16 +27,20 @@
   "dependencies": {
     "@js-temporal/polyfill": "^0.4.0",
     "html-to-text": "^8.1.0",
+    "qr-code-styling": "^1.6.0-rc.1",
+    "qrcode": "^1.5.0",
     "rrule": "^2.6.9",
-    "rss": "^1.2.2"
+    "rss": "^1.2.2",
+    "svelte-qrcode": "^1.0.0"
```

Comment:
Based on the rest of the changes in your PR, I can tell we ended up just using the `qrcode` package. So, you can go ahead and run `npm uninstall qr-code-styling svelte-qrcode` since they are unused dependencies.